### DISCUSSION TITLE
tukifporno.com - Manipulasi history apakah termasuk annoyance?

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1517] [Q&A] [answered]
https://tukifporno.com/xxx-video/abg-mesum-di-semak-semak/
Apakah ini termasuk annoyances/pop up
![20230816_211320_rmscr.jpg](https://github.com/realodix/AdBlockID/assets/114223791/18cfa08c-1a2a-4b3a-b88e-d00b1e50c247)

Putar vidio beberapa detik terus klik tombol kembali paling bawah pada android, maka bukan kembali ke halaman sebelumnya ,malah di direct ke halaman screenshot
...

**Comment by realodix** [comment, marked as answer]
Yes, itu bisa bisa dikategorikan sebagai annoyance
